lib/diagnostic.rb

```diff
@@ -8,20 +8,26 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
+require relative = ('lib/example.rb')
+
 Diagnostic.script = 'your response here'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
+pry
+
 Diagnostic.repl = 'your response here'
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
+star_wars_variable = "Star wars episode viii"
+
```

Comment:
You had the right idea here so I'll give you credit but we wanted the title of the movie to be the variable: `star_wars_episode_viii`